components/lib/multiselect/multiselect.d.ts

```diff
@@ -74,64 +74,65 @@ interface MultiSelectFilterOptions {
 }
 
 export interface MultiSelectProps extends Omit<React.DetailedHTMLProps<React.InputHTMLAttributes<HTMLDivElement>, HTMLDivElement>, 'onChange' | 'ref'> {
+    appendTo?: MultiSelectAppendToType;
+    ariaLabelledBy?: string;
+    children?: React.ReactNode;
+    className?: string;
+    dataKey?: string;
+    disabled?: boolean;
+    display?: MultiSelectDisplayType;
+    dropdownIcon?: IconType<MultiSelectProps>;
+    emptyFilterMessage?: MultiSelectEmptyFilterMessageType;
+    filter?: boolean;
+    filterBy?: string;
+    filterLocale?: string;
+    filterMatchMode?: string;
+    filterPlaceholder?: string;
+    filterTemplate?: MultiSelectFilterTemplateType;
+    fixedPlaceholder?: boolean;
     id?: string;
+    inputId?: string;
     inputRef?: React.Ref<HTMLSelectElement>;
+    itemTemplate?: MultiSelectItemTemplateType;
+    maxSelectedLabels?: number;
     name?: string;
-    value?: any;
-    options?: SelectItemOptionsType;
-    optionLabel?: string;
-    optionValue?: string;
     optionDisabled?: MultiSelectOptionDisabledType;
-    optionGroupLabel?: string;
     optionGroupChildren?: string;
+    optionGroupLabel?: string;
     optionGroupTemplate?: MultiSelectOptionGroupTemplateType;
-    display?: MultiSelectDisplayType;
```

Comment:
I hope merge conflicts with #2744 are not going to be too complex again.

Reply:
I just merged yours. Its a great feature and I will rebase mine.